src/Symfony/Component/Serializer/Normalizer/BackedEnumNormalizer.php

```diff
@@ -20,8 +20,15 @@
  *
  * @author Alexandre Daubois <alex.daubois@gmail.com>
  */
-final class BackedEnumNormalizer implements NormalizerInterface, DenormalizerInterface, CacheableSupportsMethodInterface
+final class BackedEnumNormalizer implements NormalizerInterface, DenormalizerInterface, CacheableSupportsMethodInterface, SupportedTypesMethodInterface
 {
+    public function getSupportedTypes(): ?array
+    {
+        return [
+           \BackedEnum::class  => $this->hasCacheableSupportsMethod(),
```

Comment:
double space before `=>`

since we deprecate `CacheableSupportsMethodInterface`, we should also deprecate each `hasCacheableSupportsMethod` methods individually (and not call them anymore since they should trigger a deprecation)

Reply:
done for the classes where I implemented `getSupportedTypes()`.
For the other classes, I'm waiting to see the direction we take regarding other comments/questions